### ISSUE TITLE
contentInset takes effect only after clicking annotation

### ISSUE THREAD

**Issue by CptFabulouso** [nitaliano/react-native-mapbox-gl#573]
When I set the contentInset it does not work until I click any of annotations. So when the map is first rendered, the Mapbox logo is not moved according to contentInset until annotation is clicked, then it moves to correct place.

Using version react-native-mapbox-gl 5.1.0

**Comment by CptFabulouso** [comment]
I solved this by changing file in node_modules\react-native-mapbox-gl\android\src\main\java\com\mapbox\reactnativemapboxgl\ReactNativeMapboxGLView.java

on line 149 in onMapReady method I replaced 
`_map.setPadding(_paddingLeft, _paddingTop, _paddingRight, _paddingBottom);`
with new method
`setMapPadding();`

this method does force relayout the same way when annotation is pressed
```
private void setMapPadding(){
        _map.setPadding(_paddingLeft, _paddingTop, _paddingRight, _paddingBottom); 
        // Due to a bug, we need to force a relayout on the _mapView
        _handler.post(new Runnable() {
            @Override
            public void run() {
                _mapView.measure(
                        View.MeasureSpec.makeMeasureSpec(_mapView.getMeasuredWidth(), View.MeasureSpec.EXACTLY),
                        View.MeasureSpec.makeMeasureSpec(_mapView.getMeasuredHeight(), View.MeasureSpec.EXACTLY));
                _mapView.layout(_mapView.getLeft(), _mapView.getTop(), _mapView.getRight(), _mapView.getBottom());
            }
        });
    } 
```

and finally I also changed a line in setContentInset method to look like this
```
public void setContentInset(int top, int right, int bottom, int left) {
        if (top == _paddingTop &&
            bottom == _paddingBottom &&
            left == _paddingLeft &&
            right == _paddingRight) { return; }
        _paddingTop = top;
        _paddingRight = right;
        _paddingBottom = bottom;
        _paddingLeft = left;
        if (_map != null) { 
            setMapPadding();
        }
    }
```

I have a feeling setMapPadding() is not called here, but I am not sure

**Comment by CptFabulouso** [comment]
it is fixed in version 5.2.0 (npm install will install version 5.1.0)
